src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,205 @@
+        insertDrivers(car);
+        return car;
+    }
+
+    private void deleteRelations(Car car) {
+        String query = "DELETE FROM taxi_db.cars_drivers WHERE car_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement deleteRelations = connection.prepareStatement(query)) {
+            deleteRelations.setLong(1, car.getId());
+            deleteRelations.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't delete relations from DB with that car "
+                    + car, e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String query = "UPDATE cars SET is_deleted = TRUE WHERE id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement deleteCar = connection.prepareStatement(query)) {
+            deleteCar.setLong(1, id);
+            return deleteCar.executeUpdate() != 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't delete car from DB with that id: " + id, e);
+        }
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        List<Car> allCars = new ArrayList<>();
+        String query = "SELECT c.id AS car_id, model, m.id AS manufacturer_id, m.name, m.country "
+                + "FROM cars c "
+                + "JOIN manufacturers m "
+                + "ON c.manufacturer_id = m.id "
+                + "JOIN cars_drivers cd "
+                + "ON c.id = cd.car_id "
+                + "WHERE cd.driver_id = ? AND c.is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getAllCars = connection.prepareStatement(query)) {
+            getAllCars.setLong(1, driverId);
+            ResultSet resultSet = getAllCars.executeQuery();
+            while (resultSet.next()) {
+                allCars.add(parseCar(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all from DB with driver id: "
+                    + driverId, e);
+        }
+        return allCars;
+    }
+
+    private Car parseCar(ResultSet resultSet) throws SQLException {
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setId(resultSet.getObject("manufacturer_id", Long.class));
+        manufacturer.setName(resultSet.getString("name"));
+        manufacturer.setCountry(resultSet.getString("country"));
+        Car car = new Car();
+        car.setModel(resultSet.getString("model"));
+        car.setManufacturer(manufacturer);
+        car.setId(resultSet.getObject("car_id", Long.class));
+        return car;
+    }
+
+    private List<Driver> getDriver(Long id) {
+        String query = "SELECT id, name, license_number "
+                + "FROM taxi_db.drivers d "
+                + "JOIN cars_drivers cd "
```

Comment:
come on or write the prefix "taxi_db." everywhere or nowhere. I recommend the second option